src/components/Gradebook/index.jsx

```diff
@@ -571,6 +599,7 @@ export default class Gradebook extends React.Component {
                     <small className="form-text text-muted search-help-text">Search by username, email, or student key</small>
                   </div>
                 </div>
+                {FilterBadges(this.props, this.handleFilterBadgeClose)}
```

Comment:
It's generally considered bad form to pass the entire `props` object to a child. One of the goals when writing React code is to be as explicit as possible about which props are being passed down the component hierarchy, so that components are as independent of one another as possible.